build/appveyor.yml

```diff
@@ -29,7 +31,7 @@ install:
 
     gitversion /output BuildServer
 
-    npm install
+    npm install -- --env.release=%release% --env.basename=%basename% --env.semver=%GitVersion_FullSemVer% --env.info_version=GitVersion_InformationalVersion
```

Comment:
It is not, it has been there all the time, I tried using the env variables in webpack without these arguments and that failed, so that is why I put them back in this commit

Reply:
nvm, im an idiot